rules-java/api/src/main/java/org/jboss/windup/rules/apps/mavenize/pom.xml.ftl

```diff
@@ -0,0 +1,187 @@
+
+    </#if>
+    <#list pom.dependencies>
+    <dependencies>
+        <#items as dep>
+        <dependency>
+            <groupId>${dep.groupId}</groupId>
+            <artifactId>${dep.artifactId}</artifactId>
+            <#if dep.version??>${i3}<version>${dep.version}</version>${"\n"}</#if><#t>
+            <#if dep.classifier!?trim?has_content>${i3}<classifier>${dep.classifier}</classifier>${"\n"}</#if><#t>
+            <#if (dep.packaging!"jar") != "jar">${i3}<type>${dep.packaging!"jar"}</type>${"\n"}</#if><#t>
+            <#if (dep.scope!"compile") != "compile">${i3}<scope>${dep.scope}</scope>${"\n"}</#if><#t>
+        </dependency>
+        </#items>
+    </dependencies>
+    </#list>
+
+    <#if pom.isRoot()>
+    <!-- Activate JBoss Product Maven repository.
+
+        NOTE: Configuring the Maven repository in the pom.xml file is not a recommended procedure
+        and is only done here to make it easier.
+        See the section entitled 'Use the Maven Repository' in the Development Guide for Red Hat JBoss EAP:
+        https://access.redhat.com/documentation/en/jboss-enterprise-application-platform/
+    -->
+    <repositories>
+        <repository>
+            <id>jboss-developer-staging-repository</id>
+            <url>http://jboss-developer.github.io/temp-maven-repo/</url>
+        </repository>
+        <repository>
+            <id>jboss-enterprise-maven-repository</id>
+            <url>https://maven.repository.redhat.com/ga/</url>
+        </repository>
+    </repositories>
+
+    <pluginRepositories>
+        <pluginRepository>
+            <id>jboss-developer-staging-repository</id>
+            <url>http://jboss-developer.github.io/temp-maven-repo/</url>
+        </pluginRepository>
+        <pluginRepository>
+            <id>jboss-enterprise-maven-repository</id>
+            <url>https://maven.repository.redhat.com/ga/</url>
+        </pluginRepository>
+    </pluginRepositories>
+
+    <build>
+        <plugins>
+            <!-- The WildFly plug-in deploys your EAR to a local JBoss EAP container. -->
+            <plugin>
+                <groupId>org.wildfly.plugins</groupId>
+                <artifactId>wildfly-maven-plugin</artifactId>
+                <version>1.0.2.Final</version>
+                <configuration>
+                    <!-- Due to Maven's lack of intelligence with EARs we need to skip deployment for all modules.
+                         We then enable it specifically in the EAR module. -->
+                    <skip>true</skip>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    </#if>
+    <#if pom.coords.packaging = "ear">
+    <build>
+        <finalName>${"$"}{project.parent.artifactId}</finalName>
+        <plugins>
+            <!-- EAR plug-in -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-ear-plugin</artifactId>
+                <version>2.10.1</version>
```

Comment:
it could be done in the root of the project in `pluginManagement` section and then all submodules will inherit versions and config. But no big deal if that isn't fixed t right now ;)